docs/source/usecases/SoA-and-AoSoA-with-Cabana.md

```diff
@@ -52,11 +51,11 @@ class AoSoA;
 `DataTypes`
 : The types of the elements stored in the underlying `Cabana::SoA`s.
 
-`DeviceType`
-: The Kokkos device type that carries the information about where to execute code and where to allocate storage.
+`MemorySpace`
+: The Kokkos memory space that carries information about where to allocate storage.
 
 `VectorLength`
-: The vector length for the structure of arrays (optional).
+: The vector length for the structure of arrays (optional). If not specified, a default defined for each each memory space is used; this value will likely need to be modified for optimal performance.
```

Comment:
```suggestion
: The vector length for the structure of arrays (optional). If not specified, a default defined value is used for each memory space; this value will likely need to be modified for optimal performance.
```

Reply:
```suggestion
: The vector length for the structure of arrays (optional). If not specified, a default (defined per memory space) is used; this value may need to be modified for optimal performance.
```
Thanks for catching the mistake - I also softened the language since it's strongly kernel dependent